tests/php/Unit/Utils/AuthenticatorTest.php

```diff
@@ -213,6 +215,8 @@ public function testRegisterReturnsUserObject() {
 					'user_login' => 'test',
 					'user_pass'  => 'thisisrandompass',
 					'user_email' => 'test@example.com',
+					'first_name' => 'Test',
+					'last_name'  => 'User',
```

Comment:
Add a test where name and last name both are empty. Also another test where one of these will be empty.